sdk/digitaltwins/Azure.DigitalTwins.Core/tests/TestUrlSanitizer.cs

```diff
@@ -8,14 +8,7 @@ namespace Azure.DigitalTwins.Core.Tests
 {
     internal class TestUrlSanitizer : RecordedTestSanitizer
     {
-        private const string FAKE_HOST = "fakeHost.api.wus2.digitaltwins.azure.net";
-
-        public override string SanitizeVariable(string variableName, string environmentVariableValue) =>
-            variableName switch
-            {
-                "DIGITALTWINS_URL" => SanitizeUri(environmentVariableValue),
-                _ => base.SanitizeVariable(variableName, environmentVariableValue)
-            };
+        internal const string FAKE_HOST = "fakeHost.api.wus2.digitaltwins.azure.net";
```

Comment:
FakeHost?

We don't use screaming snake case elsewhere in our code.

Reply:
I've used the existing constant name https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/digitaltwins/Azure.DigitalTwins.Core/tests/TestUrlSanitizer.cs#L11